browser/net/brave_network_audit_allowed_lists.h

```diff
@@ -63,6 +63,20 @@ constexpr const char* kAllowedUrlPrefixes[] = {
     "https://rewards-stg.bravesoftware.com/publishers/prefix-list",
     "https://grant.rewards.bravesoftware.com/v1/promotions",
 
+    // Brave Ads production
+    "https://anonymous.ads.brave.com",
+    "https://geo.ads.brave.com",
+    "https://mywallet.ads.brave.com",
+    "https://search.anonymous.ads.brave.com",
+    "https://static.ads.brave.com",
+
+    // Brave Ads staging & dev
+    "https://anonymous.ads.bravesoftware.com",
+    "https://geo.ads.bravesoftware.com",
+    "https://mywallet.ads.bravesoftware.com",
+    "https://search.anonymous.ads.bravesoftware.com",
+    "https://static.ads.bravesoftware.com",
+
```

Comment:
is it expected that the browser will make connections to these domains BEFORE the user has opted into ads? the purpose of this list is to check connections the browser makes by default.

Reply:
I thought that we should start Brave Ads after https://github.com/brave/brave-core/pull/18938 is merged.
But I was wrong. Actually, there is a bug that is a regression from https://github.com/brave/brave-core/pull/18938 - we are starting Brave Ads even if Brave Rewards are disabled. Fixed it here.

Reply:
glad to hear this file is doing its job!